### PR TITLE
Fix display issues on mobile

### PR DESCRIPTION
**Pull request by Zenoo**
Before:

After:

Before:

After:

Before:

After:

Added an option `plugin_habits_trim` to trim unused hours.  
Before:

After:

Before:

After:

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -379,6 +379,11 @@ inputs:
     description: Display coding habits charts based on recent activity
```
Comment (lowlighter) [inline action.yml:379]: This file will be auto-generated automatically, no need to edit it 👍

**Review by lowlighter — COMMENTED** [review]
Looks good to me 👍 
Can you rebase it to include the changes you made in #371 to solve conflicting files?

File: action.yml (hunk 1 above)
Comment (Zenoo) [reply action.yml:379]: My bad, fixing it now

**Comment by Zenoo** [comment]
I merged the branch :)

**Review by lowlighter — APPROVED** [review]

**Pull request merged**